docs/builtin_elements.md

```diff
@@ -41,6 +41,7 @@ or smaller. The initial width can be controlled with the `preferred-width` prope
 ### Properties
 
 * **`title`** (*string*): The window title that is shown in the title bar.
+* **`icon`** (*image*): The window icon shown in the title bar and the task bar.
```

Comment:
```suggestion
* **`icon`** (*image*): The window icon shown in the title bar and the task bar. This is only available on Windows and the X11 and Wayland windowing systems.
```

Reply:
The property is available everywhere, but it only is shown on supported system

Reply:
Ah yes, I'll edit the suggestion to reflect that. My point is that I think that it's worth mentioning limitations like that.